README.md

```diff
@@ -1,16 +1,23 @@
-# WAX Full Stack Boilerplate
+# WAX WebApp Boilerplate
+<div align="center">
+	<a href="https://edenia.com">
+		<img src="https://user-images.githubusercontent.com/5632966/178800854-cffb01ea-b55a-44b2-bcfa-872301874a43.png" width="300">
```

Comment:
lets replace the `img src` URL with: https://raw.githubusercontent.com/edenia/.github/master/.github/workflows/images/edenia-logo.png